tests/functional_tests/iterator_tests/test_toy_connectivity.py

```diff
@@ -273,7 +372,7 @@ def shift_shift_stencil2(inp):
 
 @fundef
 def shift_sparse_stencil2(inp):
-    return deref(shift(1, 3)(shift(V2E)(inp)))
+    return deref(shift(3, 1)(shift(V2E)(inp)))
```

Comment:
Please check this change carefully! Is it correct now? The change is here https://github.com/GridTools/gt4py/pull/840/files#diff-cb3ad20072297c872c4fcb6245750c91cc9aa6080da28e19b1495f78e3d9b67cR374

Reply:
It looks correct to me but the link seems broken and it just points to this PR

Reply:
Search for the change `tag_stack.pop(0)` to `tag_stack.pop()`

Reply:
Yep, still looks correct to me.